### DISCUSSION TITLE
Download Links for Bootice?

### DISCUSSION THREAD

**Discussion by kris3713** [ScoopInstaller/Extras#9545] [Q&A] [answered]
I want to add Bootice to my own bucket, the problem is I don't know which website is trustworthy enough to download the software from? I know MajorGeeks has a good reputation, but I'm not sure about FileHippo, FileHorse, Softpedia, or Softonic, as those websites look very sketchy to me. Likewise, I don't want to risk accidentally downloading malware in my system.

**Comment by HUMORCE** [comment, marked as answer]
This is an old topic, and while you may have already addressed this, here is my answer:

you can download it from a reliable distribution and host it yourself for distribution again, you will need to check distribution-related agreements of license.
